public/docs/dart/latest/tutorial/toh-pt5.jade

```diff
@@ -48,648 +48,658 @@ p Run the #[+liveExampleLink2('', 'toh-5')] for this part.
 figure.image-display
-    img(src='/resources/images/devguide/toh/mini-hero-detail.png' alt="Mini Hero Detail" height="70")
+  img(src='/resources/images/devguide/toh/mini-hero-detail.png' alt="Mini Hero Detail" height="70")
 :marked
-    ### Format with the *UpperCasePipe*
+  ### Format with the *UpperCasePipe*
 
-    Notice that the hero's name is displayed in CAPITAL LETTERS. That's the effect of the `UpperCasePipe`
-    that we slipped into the interpolation binding. Look for it right after the pipe operator, ( | ).
+  Notice that the hero's name is displayed in CAPITAL LETTERS. That's the effect of the `UpperCasePipe`
+  that we slipped into the interpolation binding. Look for it right after the pipe operator ( | ).
 +makeExample('toh-5/dart/lib/heroes_component.html','pipe')(format=".")
 :marked
-    Pipes are a good way to format strings, currency amounts, dates and other display data.
-    Angular ships with several pipes and we can write our own.
+  Pipes are a good way to format strings, currency amounts, dates and other display data.
+  Angular ships with several pipes and we can write our own.
 .l-sub-section
-    :marked
-        Learn about pipes in the [Pipes](../guide/pipes.html) chapter.
-:marked
-    ### Move content out of the component file
-    We are not done. We still have to update the component class to support navigation to the
-    `HeroDetailComponent` when the user clicks the *View Details* button.
-
-    This component file is really big. Most of it is either template or CSS styles.
-    It's difficult to find the component logic amidst the noise of HTML and CSS.
-
-    Let's migrate the template and the styles to their own files before we make any more changes:
-    1. *Cut-and-paste* the template contents into a new `heroes_component.html` file.
-    1. *Cut-and-paste* the styles contents into a new `heroes_component.css` file.
-    1. *Set* the component metadata's `templateUrl` and `styleUrls` properties to refer to both files.
-
-    The revised component data looks like this:
+  :marked
+    Learn about pipes in the [Pipes](../guide/pipes.html) chapter.
+:marked
+  ### Move content out of the component file
+  We are not done. We still have to update the component class to support navigation to the
+  `HeroDetailComponent` when the user clicks the *View Details* button.
+
+  This component file is really big. Most of it is either template or CSS styles.
+  It's difficult to find the component logic amidst the noise of HTML and CSS.
+
+  Let's migrate the template and the styles to their own files before we make any more changes:
+  1. *Cut-and-paste* the template contents into a new `heroes_component.html` file.
+  1. *Cut-and-paste* the styles contents into a new `heroes_component.css` file.
+  1. *Set* the component metadata's `templateUrl` and `styleUrls` properties to refer to both files.
+  
+  We no longer display the `HeroDetailComponent` in the `HeroesComponent` template because we're navigating to it.
```

Comment:
Let's reword this a bit to clarify what "it" is. Also, maybe you could show the imports, since you mention them? Or don't show them, but don't mention them. (If you do mention them, try to reword "might as well".) Maybe:

Because the template for `HeroesComponent` no longer _uses_ `HeroDetailComponent` directly &mdash;
instead using the router to _navigate_ to it &mdash; we can remove `HeroDetailComponent` from the `directives` list. That list is now empty, so we can remove the `directives` argument. The revised `@Component` looks like this: